### PR TITLE
chore: add disclaimer to README.md

### PR DESCRIPTION
**Pull request by damiannolan**
- Adding disclaimer to README.md

### CODE REVIEW

**Comment by damiannolan** [comment]
Suggestions welcome

File: README.md [hunk 1]
```diff
@@ -1,18 +1,23 @@
...
+
 ## Overview 
 
 The following repository contains a basic example of an Interchain Accounts authentication module and serves as a developer guide for teams that wish to use interchain accounts functionality.
 
-The Interchain Accounts module is currently under active development and has been moved to the `ibc-go` repository [here](https://github.com/cosmos/ibc-go/tree/main/modules/apps/27-interchain-accounts). Interchain Accounts is aiming to release in Q1 2022.
-
-This repository will be updated regularly to consume pre-release versions of `ibc-go` equipped with the Interchain Accounts module.
+The Interchain Accounts module is now maintained within the `ibc-go` repository [here](https://github.com/cosmos/ibc-go/tree/main/modules/apps/27-interchain-accounts). 
+Interchain Accounts is now included in the [`v3.0.0`](https://github.com/cosmos/ibc-go/releases/tag/v3.0.0) release of `ibc-go`.
```
Comment (damiannolan) [inline README.md:14]:
```suggestion
Interchain Accounts is now available in the [`v3.0.0`](https://github.com/cosmos/ibc-go/releases/tag/v3.0.0) release of `ibc-go`.
```

File: README.md [hunk 2]
```diff
@@ -1,18 +1,23 @@
 # Interchain Accounts
 
+### Disclaimer
+
+The following repository and [`x/inter-tx`](./x/inter-tx/) module serves as an example and is used to exercise the functionality of Interchain Accounts end-to-end for development purposes only.
+This module *SHOULD NOT* be used in production systems and developers building on Interchain Accounts are encouraged to design their own authentication modules which fit their use case.
```
Comment (crodriguezvega) [inline README.md:6]:
```suggestion
This module **SHOULD NOT** be used in production systems and developers building on Interchain Accounts are encouraged to design their own authentication modules which fit their use case.
```

File: README.md [hunk 3]
```diff
@@ -1,18 +1,23 @@
 # Interchain Accounts
 
+### Disclaimer
```
Comment (crodriguezvega) [inline README.md:3]:
```suggestion
## Disclaimer
```

**Review by crodriguezvega — APPROVED** [review]

**Pull request merged**